LDM-294.tex

```diff
@@ -15,7 +15,8 @@
 \addtohist{3.5}{2018-06-18}{New DM product tree and revised org chart. Add deputy subsystem scientist. Approved in \jira{RFC-493}}{J.~Swinbank}
 \addtohist{3.6}{2019-01-29}{Review product tree. Add release management subsection. Update DMCCB and DMSE team. Approved in \jira{RFC-561}}{G.~Comoretto}
 \addtohist{3.7}{2019-02-06}{Add text for LSP and Middleware mangers. Time allocation policy for institutional scientists, add admin names to org chart. Approved in \jira{RFC-572}}{W.~O'Mullane, L.~Guy}
-\addtohist{}{2019-07-12}{Org charts, glossary, 3rd party software, risk management, milestones.}{W.~O'Mullane, L.~Guy}
+\addtohist{3.8}{2019-07-12}{Org charts, glossary, 3rd party software, risk management, milestones.}{W.~O'Mullane, L.~Guy}
+\addtohist{}{2019-12-21}{SST org charts, middleware lead,  milestones. \jira{DM-22393}{W.~O'Mullane}
```

Comment:
Missing curly brace after the `\jira{}`.

Reply:
got it